types/types.go

```diff
@@ -111,3 +113,32 @@ func UnpackWarpMessage(unsignedMsgBytes []byte) (*avalancheWarp.UnsignedMessage,
 	}
 	return unsignedMsg, nil
 }
+
+func LogsToBlocks(logs []types.Log) ([]*WarpBlockInfo, error) {
+	slices.SortFunc(logs, func(a, b types.Log) int {
+		if a.BlockNumber < b.BlockNumber {
+			return -1
+		}
+		if a.BlockNumber > b.BlockNumber {
+			return 1
+		}
+		return 0
+	})
```

Comment:
`sort.SliceStable` which uses a boolean sortfunc is way more concise

Reply:
Yeah that's the way I had it originally but swapped because of a IDE hint saying that slices.SortFunc is faster/more readable.. Not sure that I agree on the more readable.

Reply:
lol I feel like you're being gaslit 😛